libraries/AP_RangeFinder/AP_RangeFinder_PulsedLightLRF.cpp

```diff
@@ -140,6 +146,17 @@ static const struct settings_table settings_v2[] = {
     { LL40LS_MEASURE_REG, LL40LS_MSRREG_ACQUIRE },
 };
 
+/*
+  register setup table for V3HP Lidar
+ */
+static const struct settings_table settings_v3hp[] = {
+    { LL40LS_MEASURE_REG, LL40LS_MSRREG_ACQUIRE },
+    { LL40LS_SIG_COUNT_MAX, 0x80},
```

Comment:
Was this value by experimentation, because it was the v3 default value or any other reason? Likely it is actually correct, unfortunately the datasheet gives a unit-less minimum rate related to this value.

Reply:
Hi, I got this value (0x80: 128 acquisitions) from Arduino example code for balanced performance configuration. V3HP default is 0xFF (255 acquisitions-maximum possible) but in practice I don't see any change in performance.